netbox/extras/models/scripts.py

```diff
@@ -22,12 +23,50 @@
 logger = logging.getLogger('netbox.data_backends')
 
 
-class Script(EventRulesMixin, models.Model):
-    """
-    Dummy model used to generate permissions for custom scripts. Does not exist in the database.
-    """
+class Script(EventRulesMixin, JobsMixin, models.Model):
+    name = models.CharField(
+        verbose_name=_('name'),
+        max_length=79,
+    )
+    module = models.ForeignKey(
+        to='extras.ScriptModule',
+        on_delete=models.CASCADE,
+        related_name='scripts'
+    )
+    is_valid = models.BooleanField(
```

Comment:
We might want to rename this for clarity as "valid" is pretty ambiguous. I think specifically we want to convey whether the script can be executed; is that correct? Maybe something like `is_executable` or `can_run`?